app/models/hub_edos/response_handler.rb

```diff
@@ -2,14 +2,23 @@ module HubEdos
   module ResponseHandler
 
     def unwrap_response(response)
-      return [] unless response.is_a? Hash
-      unwrapped = wrapper_keys.inject(response) { |feed, key| (feed.is_a?(Hash) && feed[key]) || [] }
-      unwrapped.respond_to?(:each) ? unwrapped : []
```

Comment:
For these unexpected cases I think we should raise errors, to aid in detection in DEV/QA. Such is an exceptional situation, not expected.